scripts/Load_collection/load_collection.yml

```diff
@@ -0,0 +1,64 @@
+script: load_collection.R
+name:  load collection to calculate metric
+description: "This code allows loading a collection containing information for metric calculation. The information is filtered by the WKT-generated study area coordinates."
+author: 
+ - name: Maria Alejandra Molina Berbeo (mmolina@humboldt.org.co)
+ - name: Victor Julio Rincon (vrincon@humboldt.org.co)
+ - name: Luis Fernando Urbina (lurbina@humboldt.org.co)
+external_link: https://github.com/GEO-BON/biab-2.0
+
+inputs:
+  collection_path:
+    label: collection path
+    description: Path where the collection is hosted. The path can be hosted on the BIAB server (subject to approval from administrators) or in a cloud container
+    type: text
+    example: 'scripts/Load_collection/Collections/Colombia_human_footprint_collection'
```

Comment:
Is this accessed as a STAC catalog or a bunch of tiffs? If STAC then it could be put on some object storage and receive a path to the stac as an input.